File: 릿코드/0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer, Integer> map = new HashMap<>();
        for(int i=0, len=nums.length; i<len; i++) {
            if(map.get(target-nums[i]) != null) return new int[]{map.get(target-nums[i]), i};
            map.put(nums[i], i);
        }
        return null;
    }
}

File: 릿코드/0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]: 
        for i, vI in enumerate(nums):
            for j, vJ in enumerate(nums):
                if i == j: continue
                if vI+vJ == target: return [i,j]
        